docs/admin/troubleshooting/system-tables.rst

```diff
@@ -120,8 +121,11 @@ To list all nodes using more than 98 per cent of the memory, type::
     ...
     SELECT ... in set (... sec)
 
-The table also contains the performance metrics like the load average, disk,
-memory, heap, or network throughput. Running::
+The table also contains performance metrics like the load average, disk,
+memory, heap, or network throughput.
+The object has the same structure as the **_node** system column of
```

Comment:
Can we fix the `sys.operations` query right now please as well? (it should be: `SELECT node['name'], node['id'], * FROM sys.operations;`
```suggestion
The object has the same structure as the **node** system column of
```